stubs/emoji/emoji/core.pyi

```diff
@@ -26,22 +22,13 @@ def emojize(
 ) -> str: ...
 def demojize(
     string: str,
-    use_aliases: bool | type[_DeprecatedParameter] = ...,
     delimiters: tuple[str, str] = ...,
     language: str = ...,
     version: float | None = ...,
     handle_version: str | Callable[[str, dict[str, str]], str] | None = ...,
 ) -> str: ...
-def replace_emoji(
-    string: str,
-    replace: str | Callable[[str, dict[str, str]], str] = ...,
-    language: str | type[_DeprecatedParameter] = ...,
-    version: float | None = ...,
-) -> str: ...
-def get_emoji_regexp(language: str | None = ...) -> Pattern[str]: ...
-def emoji_lis(string: str, language: str | type[_DeprecatedParameter] = ...) -> list[_EmojiLisReturn]: ...
+def replace_emoji(string: str, replace: str | Callable[[str, dict[str, str]], str] = ..., version: float | None = ...) -> str: ...
```

Comment:
I think the existing annotation here was incorrect: it can't be `None`, because it's compared with `-1` here: https://github.com/carpedm20/emoji/blob/c398e0c5ff6ceff08c7860078b242765f093f285/emoji/core.py#L228

```suggestion
def replace_emoji(string: str, replace: str | Callable[[str, dict[str, str]], str] = ..., version: float = ...) -> str: ...
```

Reply:
Ah, good point. The `| None` is only on the downstream functions which this is acting as a wrapper for.